src/features/embed-gist-inline.js

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+
+const gistRegex = /gist\.github\.com\/\w*?\/\w*/;
+const isGist = link => gistRegex.test(link.href);
+
+const getJsonUrl = href => `${href}.json`.replace(/\.jso?n?\.json$/, '.json');
+const getGistData = href => fetch(getJsonUrl(href)).then(response => response.json());
+
+const createGistElement = gistData => {
+  const el = document.createElement('div');
+  const style = `<link rel="stylesheet" href="https://assets-cdn.github.com/assets/gist-embed-3cc724162479db25e452fdf621f2349adef3e742b53552c2a93f82d28156cb96.css" />`;
+  el.innerHTML = style + gistData.div;
```

Comment:
Do we need to do any sanitizing here?

Reply:
The data from the gist is sanitised already, are we planning in case a gist response is somehow compromised?

Reply:
I don't know. I just asked in case anyone could think of a case. I guess it's fine then.